app/controllers/application_controller.rb

```diff
@@ -1583,6 +1583,7 @@ def get_view(db, options = {})
       :association           => association,
       :filter                => get_view_filter(options[:filter]),
       :sub_filter            => get_view_process_search_text(view),
+      :supports_filter       => options[:supports_filter],
```

Comment:
I find the name `supports_filter` colliding with the definition of features. If you are used to read `supports :feature` in the code, this would read to: "this thing supports filter"
But its really a filter for supported features. How about naming this
`:supported_features_filter` or `:filter_for_supported_features` ?
This also helps grepability.